tests/unit/steps/test_steps_analyze.py

```diff
@@ -133,6 +136,19 @@ async def test_analysis_detect_current_cloud_os_release_same_release(apps, model
 
     # current_cloud_os_release takes the minimum OpenStack version
     assert result.current_cloud_os_release == "ussuri"
+    assert result.next_cloud_os_release == "victoria"
+
+
+@pytest.mark.asyncio
+async def test_analysis_detect_current_and_next_cloud_os_release_empty_cloud(model):
```

Comment:
not-blocking: I think that using docstring in test function make are more readable than long name of function.